src/component/Calculator.js

```diff
@@ -1,44 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './style/Calculator.css';
-import Display from './Display';
+import calculate from '../Logic/calculate';
 
-const Calculator = () => (
-  <div id="calculator">
-    <Display />
-    <div id="keypad">
-      <button type="button">AC</button>
-      <button type="button">+/-</button>
-      <button type="button">%</button>
-      <button type="button" className="orange">
-        ÷
-      </button>
-      <button type="button">7</button>
-      <button type="button">8</button>
-      <button type="button">9</button>
-      <button type="button" className="orange">
-        *
-      </button>
-      <button type="button">4</button>
-      <button type="button">5</button>
-      <button type="button">6</button>
-      <button type="button" className="orange">
-        -
-      </button>
-      <button type="button">1</button>
-      <button type="button">2</button>
-      <button type="button">3</button>
-      <button type="button" className="orange">
-        +
-      </button>
-      <button type="button" className="zero">
-        0
-      </button>
-      <button type="button">.</button>
-      <button type="button" className="orange">
-        =
-      </button>
+const Calculator = () => {
+  const [total, setTotal] = useState(null);
+  const [next, setNext] = useState(null);
+  const [operation, setOperation] = useState(null);
+  const handleClick = (e) => {
+    const result = calculate({ total, next, operation }, e.target.innerText);
+    setTotal(result.total);
+    setNext(result.next);
+    setOperation(result.operation);
+  };
+  return (
+    <div id="calculator">
+      <div className="display">
+        {total}
+        {next}
+        {operation}
```

Comment:
- [Optional] Hi @QueenterJuma, I'm really excited because your app is working as expected. Let's kindly make sure operators appear between operants as in normal calculators. For example, as we can see on the screen record below, we should expect **77 x 9** instead of **779x**

[Screencast from 03-23-2023 12:39:30 AM.webm](https://user-images.githubusercontent.com/63087491/227063722-06023657-acb7-4b17-8a8d-7cb51364c5c1.webm)



- I think you just need to place `{operation}` in between   `{total}` and `{next}`